src/chrome/locale/en/https-everywhere.dtd

```diff
@@ -19,7 +19,7 @@
 <!ENTITY https-everywhere.menu.observatory "SSL Observatory Preferences">
 <!ENTITY https-everywhere.menu.globalEnable "Enable HTTPS Everywhere">
 <!ENTITY https-everywhere.menu.globalDisable "Disable HTTPS Everywhere">
-<!ENTITY https-everywhere.menu.blockHttpRequests "Block all HTTP requests">
+<!ENTITY https-everywhere.menu.blockHttpRequests "Block all unencrypted requests">
```

Comment:
I think if you want a new translation here you're going to need to create a new item.